setup.py

```diff
@@ -1,4 +1,60 @@
-from setuptools import setup
+#!/usr/bin/env python
 
-# Redefine `name` here so GitHub’s "Used By" can detect the package.
-setup(name="wagtail-footnotes")
+# from setuptools import setup
+
+# # Redefine `name` here so GitHub’s "Used By" can detect the package.
+# setup(name="wagtail-footnotes")
+
+
+from os import path
+
+from setuptools import find_packages, setup
+
+from wagtail_footnotes import __version__
+
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="wagtail-footnotes",
+    version=__version__,
+    description="Add footnotes to rich text in your Wagtail pages",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    author="Cameron Lamb",
+    author_email="cameron.lamb@torchbox.com",
+    url="https://github.com/torchbox/wagtail-footnotes",
```

Comment:
worth looking at https://github.com/wagtail/wagtail/pull/8112/files when it comes to excluding tests from packaging.

Reply:
This will be next up, thanks.

Reply:
I'm not 100% sure about these changes.

Reply:
I was seeing.

```
/Users/nickm/.pyenv/versions/wagtail-footnotes/lib/python3.8/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'wagtail_footnotes.static.footnotes.js' as data is deprecated, please list it in `packages`.
    !!


    ############################
    # Package would be ignored #
    ############################
    Python recognizes 'wagtail_footnotes.static.footnotes.js' as an importable package,
    but it is not listed in the `packages` configuration of setuptools.

    'wagtail_footnotes.static.footnotes.js' has been automatically added to the distribution only
    because it may contain data files, but this behavior is likely to change
    in future versions of setuptools (and therefore is considered deprecated).

    Please make sure that 'wagtail_footnotes.static.footnotes.js' is included as a package by using
    the `packages` configuration field or the proper discovery methods
    (for example by using `find_namespace_packages(...)`/`find_namespace:`
    instead of `find_packages(...)`/`find:`).

    You can read more about "package discovery" and "data files" on setuptools
    documentation page.


!!
```